PyPortal_ArduinoSelfTest/PyPortal_ArduinoSelfTest.ino

```diff
@@ -76,20 +77,21 @@ void setup() {
     tft.println("FAILED");
     while (1);
   }
-  uint8_t manid, devid;
+  uint8_t manid[512], devid[512];
```

Comment:
dont do this, the man and dev id are 1 byte. pass in the location by reference as in the examples

https://github.com/adafruit/Adafruit_SPIFlash/blob/cf87b775540bf882e04d5f0cc34102630cab4eac/examples/SPI_Flash_Manipulator/SPI_Flash_Manipulator.ino#L42